sdk/policyinsights/Azure.ResourceManager.PolicyInsights/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0-beta.1 (2023-05-31)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
```

Comment:
@dvbb Please help to add below for Bugs Fixed:
- Fixed an issue that exception throws when `ResourceIdentifier` type field is empty during deserialization of `PolicyDetails`.